src/appMain/life_cycle.h

```diff
@@ -45,10 +45,6 @@
 #include "hmi_message_handler/messagebroker_adapter.h"
 #endif  // #if ( defined (MESSAGEBROKER_HMIADAPTER) || defined(PASA_HMI)  )
 #include "application_manager/application_manager_impl.h"
-#ifdef SDL_REMOTE_CONTROL
```

Comment:
does it  (removing the item) mean SDL_REMOTE_CONTROL is always included in the build?